lol.janjic.Splitcat.yml

```diff
@@ -0,0 +1,48 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+---
+app-id: lol.janjic.Splitcat
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: splitcat
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+modules:
+  - name: Splitcat
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - |
+        mv splitcat /app/
+        chmod +x /app/splitcat/splitcat
+        mkdir -p /app/bin
+        ln -s /app/splitcat/splitcat /app/bin/splitcat
+
+        echo "$FLATPAK_DEST"
+
+        ls /app/splitcat
+
+        install -Dm644 /app/splitcat/$FLATPAK_ID.png -t $FLATPAK_DEST/share/icons/hicolor/256x256/apps/
+        install -Dm644 /app/splitcat/$FLATPAK_ID.desktop -t $FLATPAK_DEST/share/applications/
+        install -Dm644 /app/splitcat/$FLATPAK_ID.appdata.xml -t $FLATPAK_DEST/share/metainfo
+    sources:
+      - type: archive
+        url: https://github.com/vogonwann/splitcat/releases/download/v0.1.6/splitcat-linux-x64.tar.gz
+        dest: splitcat
+        strip-components: 0
+        sha256: 5e1052fa7581d06c2d49293a29624fffd9316a8439932e494df0cf08898d3cc8
+        only-arches:
+          - x86_64
+      - type: archive
+        url: https://github.com/vogonwann/splitcat/releases/download/v0.1.5/splitcat-linux-arm64.tar.gz
```

Comment:
the version seems different than x86_64 ?

Reply:
yes, I need to build 'em manually, because GitHub action runners are not building flutter apps for arm64 nicely. There was no changes, so I just copied the tarball from previous release.

Reply:
Starting build of lol.janjic.Splitcat
Cleaning up
FB: Mounting read-only fs: rofiles-fuse /srv/buildbot/worker/build-aarch64/build/builddir /srv/buildbot/worker/build-aarch64/build/.flatpak-builder/rofiles/rofiles-9xZae5
FB: unmounting rofiles-fuse /srv/buildbot/worker/build-aarch64/build/.flatpak-builder/rofiles/rofiles-9xZae5
Committing stage cleanup to cache
Finishing app
]2;flatpak-builder: Finishing lol.janjic.SplitcatFB: Mounting read-only fs: rofiles-fuse /srv/buildbot/worker/build-aarch64/build/builddir /srv/buildbot/worker/build-aarch64/build/.flatpak-builder/rofiles/rofiles-9xZae5
Error: Command 'splitcat' not found
FB: Unmounting read-only fs: fusermount -uz /srv/buildbot/worker/build-aarch64/build/.flatpak-builder/rofiles/rofiles-9xZae5
program finished with exit code 1
elapsedTime=280.261907

this is the error in production build. I'm not getting that one locally...